Relativity.Processing.SDK.md

```diff
@@ -4,20 +4,41 @@
 
 This package contains interfaces for the public APIs of the Relativity Processing application.
 
-## 11.3.101.10
+## 12.2.13
 
 ### Release Notes
 
-* Initial version

```

Comment:
We should probably also include a note regarding the additional of a versioned **v1** API surface and that consumers are highly encouraged to transition to those APIs.